storybook/stories/Button/IconButton.js

```diff
@@ -35,7 +35,7 @@ const IconButton = props => {
 		small: theme.iconSizeSmall,
 		xsmall: theme.iconSizeXSmall
 	};
-	const { name, size, color, onPress, backgroundColor } = props;
+	const { name, size, onPress, iconColor, backgroundColor } = props;
 
 	return (
 		<View>
```

Comment:
Is this view useful?

Reply:
Hmmm...  I can't think of any reason why it would be, no.  Admittedly blindly followed the conventional suggestion of the book I had worked with.